src/components/APIMarkdownContent/index.tsx

```diff
@@ -27,12 +29,7 @@ const APIMarkdownContent: React.FC<APIMarkdownContentProps> = ({ id }) => {
       localStorage.getItem("jwt")
     );
     try {
-      const response = await axios.get(`${url}/documents/${id}/`, {
-        headers: {
-          Authorization: `Bearer ${token.access}`,
-        },
-      });
-      setContent(response.data.content);
+      getContent(url, id, token, setContent);
```

Comment:
better getContent return content and setContent the text that getContent return

Reply:
This is all great feedback. Thanks for the time.